### ISSUE TITLE
ImportError: No module named cuda.load_textures

### ISSUE THREAD

**Issue by TheCompAce** [daniilidis-group/neural_renderer#80]
When I try to run an example I get the following error...

"import neural_renderer.cuda.load_textures as load_textures_cuda"

I have spent all day trying to get this to work. I created a new Environment with Python 3.6, and got everything setup (I also needed the Nvidia CUDA toolkit)

But now I am stuck with this error. (I tried the latest github of "neural_renderer" but that didn't compile like this one did.)

Any help with this would be great, I am on Windows 7.

EDIT:
I decided to install tensorflow-gpu instead of tensorflow.
ran "conda uninstall tensorflow"
ran "conda install tensorflow-gpu"

So now I'm getting
"File "h5py\h5.pyx", line 1, in init h5py.h5"
"ImportError: DLL load failed: The specified procedure could not be found."

so I ran "pip uninstall h5py" (I had to run it twice, for duplicate versions)
ran "conda uninstall h5py" (now my env should be clean of h6py)
rand "conda install h5py"

This fixed the h5py error.

But when I tried the demo again, so it seems that it wasn't tensorflow causing it.
"import neural_renderer.cuda.load_textures as load_textures_cuda"

**Comment by zfying** [comment]
Ran into the same problem. I'm also using tensorflow without gpu and commit 55a05a version of neural_renderer. Any luck with solving the problem?

**Comment by gaojingqian** [comment]
pip uninstall it
download the neural_renderer project and unzip it
cd neural_renderer and run pip install .
if setup correctly and still get this ImportError, cd neural_renderer \neural_renderer , run python, import neural_renderer, you will find it work well and the ImportError never comes again!!! Amazing!

**Comment by peteryuX** [comment]
For me, the problem caused by the import package name is same as existed folder name. Move to other location should solve this problem.

**Comment by CharlesNord** [comment]
I found that the import error occurs when the current directory in the folder of "neural_renderer-master." If you change to another directory, the import works fine. It seems because the package name is duplicated with the /neural_renderer-master/neural_renderer.

**Comment by pfeducode** [comment]
> 我发现当前目录在“neural_renderer-master”文件夹中时出现导入错误。如果您切换到另一个目录，则导入工作正常。似乎是因为包名与 /neural_renderer-master/neural_renderer 重复。

you are right, we should rename   Rotate-and-Render-master/neural_renderer. such as Rotate-and-Render-master/pf_neural_renderer. then your problem will solved and  the model(neural_renderer.cuda.load_textures), will be imported in your project

**Comment by WhitePhilosopher** [comment]
楼上正解，解决了我一个下午的烦恼
